specifications/active/aRMT-1.5.2.yml

```diff
@@ -0,0 +1,138 @@
+name: aRMT
+vendor: RADAR
+model: aRMT-App
+version: 1.5.2
```

Comment:
Can we version it as 1.6.0. Since we are adding topics this will be considered a minor version update and not a patch version. 
Thanks